src/pages/_id/index.vue

```diff
@@ -255,22 +255,10 @@ export default {
       return this.supportingAmount > 0;
     },
     ctaToRoute() {
-      if (this.getUserIsCivicLikerV2) {
-        if (this.isSupportingCreator) {
-          return undefined;
```

Comment:
So there is no undefined case now?

Reply:
Updated

Reply:
How about the `getUserIsCivicLikerV2` checking?